source/about.rst

```diff
@@ -0,0 +1,212 @@
+:orphan:
+
+Our Mission
+===========
+
+.. raw:: html
```

Comment:
This, and many other places should be replaced. Where possible we want to be using restructred text markup for all the content.
I appreciate this is a good stop gap for now, but we should put it on the list.